### DISCUSSION TITLE
Any way to change the default theme in "Reborn" preset to a more friendly dark mode if no cover image is found?

### DISCUSSION THREAD

**Discussion by Woof420** [TT-ReBORN/Georgia-ReBORN#133] [Q&A] [answered]
Hello.

Currently if you choose the theme "Reborn" it will adapt based on the artwork colors and I really like that behavior. It's one of the main reasons I use this theme after all. However, sometimes tracks don't have artwork embedded or it doesn't get properly embedded when in the folder. This causes the theme to default to a white/greyish theme which blinds me at night whenever this happens.

Example of the current default (no artwork) theme when selecting Reborn:
![image](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/135785312/98a3c2f2-2dee-4a29-9737-5e33d137cdf7)

![foobar2000_2023-07-05_22-14-22](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/135785312/6a0990d0-07c1-46ee-bba1-425f8148704b)

Could it be possible to change the default theme when no artwork found to a different more dark-mode friendly theme (such as the "dark blue" preset)?

Example of the ideal behavior when no artwork is found but also having Reborn selected as theme:
![image](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/135785312/46ba5697-c45d-45e6-a259-55fbe6c742e3)

Unrelated to that, I've been having issues to update the artwork tags in tracks but I think that's a foobar2k bug and not Georgia related (foobar freezes completely and I gotta force close it on task manager). The issue above is becoming more annoying as I usually fixed it by quickly updating the properties of the track tags within foobar and now that seems to be broken (I need to look up a different app to update tags in my files or solve whatever is freezing foobar2k when changing artwork)

Thanks for reading the above.

**Comment by TT-ReBORN** [comment, marked as answer]
Hi @Woof420,

if there is no album art cover to be shown, it can not extract any colors from the image.
That is why it's being displayed in white color. There is a new feature in the upcoming RC-2 update where 
you can manually override the current theme and set custom theme, styles or preset for every album via GR-tags.
More information about this feature can be found [here](https://github.com/TT-ReBORN/Georgia-ReBORN/discussions/112#discussioncomment-6206037).

---

About your tagging issue, there is no problem with Georgia-ReBORN.
The freeze could happen if your firewall is blocking foobar or any other software.
Another possibility could be if you click around and your HDD is not spinning and in idle mode ( i.e sleeping ), 
it takes time once your HDD is ready again, it could become unresponsive.
I would also try to install a new portable foobar2000 v2.0 stable ( not preview ) without any components or themes, 
just standard original foobar and test the tagging to see if the problem still occurs.

-TT

**Reply by dsandii** [reply]
First of all, thank you for your excellent work on this project. Apologies for necroing this discussion, but I'm having the exact same complaint as the OP (at least with respect to missing album art).

I love this project and really want to always use the Reborn theme when applicable but I really don't enjoy the blaring white interface whenever a file doesn't have associated album art metadata. 

The GR-tagging feature is excellent but for my purposes not ideal as I have a lot of loose MP3s and setting album tags on what's effectively a track by track basis isn't convenient or very feasible.

I was curious if there is a way to add a custom fall-back "default album art" that will then allow Reborn to extract whatever color information it needs to render a theme (instead of blinding me with all-white)?

**Reply by TT-ReBORN** [reply]
Hi @dsandii,

actually the DEV version has these implemented:

```
- Implemented a new configurable theme day/night mode and made improvements
  * Custom time range for theme day/night mode can be set up via top menu:
    Options > Settings > Theme day/night mode > Set custom time range
  * It is now possible to set up and configure day and night theme via
    top menu Options > Settings > Theme day/night mode > Theme setup for daytime
    top menu Options > Settings > Theme day/night mode > Theme setup for nighttime
  * When the theme setup for daytime/nighttime is active, you can choose your theme and styles
    or preset and also configure theme brightness if needed. Once configured, return to
    the Theme day/night mode menu and save your settings. Do this for both day and night modes
  * During theme setup, only the top menu options that are configurable are available for easier navigation
  * To activate theme day/night mode, go to top menu Options > Settings > Theme day/night mode
    and choose your preferred time range when day or night theme will change during the day
  * The default theme for day theme is White theme and for night is Black theme,
    but you can change it to whatever theme you like via the Theme day/night mode theme setup
  * The improved theme day/night mode now also supports custom GR theme tags usage
  * Reborn and Random theme will now start dark-themed if it is nighttime
```

Latest commit now also includes: `Applied Reborn/Random's "Night" theme style when no album art exists during playback`

You can download it from here:
https://github.com/TT-ReBORN/Georgia-ReBORN/archive/refs/heads/master.zip

Once you have the DEV version installed, you can then set up the theme day/night mode:
In daytime, set your `Reborn` theme **without** using top menu `Options` > `Style` > `Night` and other styles if desired. 
In nighttime, set your `Reborn` theme **with** the `Night` style and other styles if desired.

Or if you want to have `Reborn` theme to always stay dark, just choose `Options` > `Style` > `Night` 
without setting up the theme day/night mode.

-TT

**Reply by dsandii** [reply]
Excellent, thank you for the thorough response!
